tests/IntegrationTest.php

```diff
@@ -211,7 +211,7 @@
 
     $this->assertFalse($firstResponse->headers->has('X-Cached-At'));
     assertTrue($secondResponse->headers->has('X-Cached-At'));
-    $this->assertInstanceOf(DateTime::class, $secondResponse->headers->getDate('X-Cached-At'));
+    $this->assertInstanceOf(DateTimeInterface::class, $secondResponse->headers->getDate('X-Cached-At'));
```

Comment:
Depending on the version, it may create a `DateTime::class` or a `DateTimeImmutable::class`. I'm not sure why this happens either.